src/app.js

```diff
@@ -49,53 +50,11 @@ p5.prototype.setVRBackgroundColor = function(r, g, b) {
   p5xr.instance.curClearColor = color(r, g, b);
 };
 
-p5.RendererGL.prototype._update = function() {
-  /* TODO: Make a diff function to avoid overwriting this function */
-  /* IE: Override the resetting of cameraMatrices in _update */
-  /*
-    /*
-    /*
-    /*
-    /*
-    */
-
-  // reset model view and apply initial camera transform
-  // (containing only look at info; no projection).
-  // this.uMVMatrix.set(
-  //   this._curCamera.cameraMatrix.mat4[0],
-  //   this._curCamera.cameraMatrix.mat4[1],
-  //   this._curCamera.cameraMatrix.mat4[2],
-  //   this._curCamera.cameraMatrix.mat4[3],
-  //   this._curCamera.cameraMatrix.mat4[4],
-  //   this._curCamera.cameraMatrix.mat4[5],
-  //   this._curCamera.cameraMatrix.mat4[6],
-  //   this._curCamera.cameraMatrix.mat4[7],
-  //   this._curCamera.cameraMatrix.mat4[8],
-  //   this._curCamera.cameraMatrix.mat4[9],
-  //   this._curCamera.cameraMatrix.mat4[10],
-  //   this._curCamera.cameraMatrix.mat4[11],
-  //   this._curCamera.cameraMatrix.mat4[12],
-  //   this._curCamera.cameraMatrix.mat4[13],
-  //   this._curCamera.cameraMatrix.mat4[14],
-  //   this._curCamera.cameraMatrix.mat4[15]
-  // );
-  // p5.instance._renderer.uMVMatrix.set(viewMat);
-  // p5.instance._renderer.uPMatrix.set(projMat);
-  // reset light data for new frame.
-
-  /*
-    /*
-    /*
-    /*
-    /*
-    */
-  /* TODO: Make different function to avoid overwriting this function */
-  /* IE: Override the resetting of cameraMatrices in _update */
-
+p5xr.prototype._updatexr = function () {
   this.ambientLightColors.length = 0;
```

Comment:
This no longer correct. Since this is a method of p5xr now it does not have access to the lights this way. A simple way to replace this code would be something like:
```

p5xr.prototype._updatexr = function() {
  p5.instance._renderer.ambientLightColors.length = 0;
  p5.instance._renderer.directionalLightDirections.length = 0;
  p5.instance._renderer.directionalLightColors.length = 0;

  p5.instance._renderer.pointLightPositions.length = 0;
  p5.instance._renderer.pointLightColors.length = 0;
};
```
This doesn't account for running p5xr with p5-instance mode, but I don't think it makes sense to offer support for that.